Week3/homework/step2-3.js

```diff
@@ -17,8 +23,17 @@ console.log('for', repeatStringNumTimesWithFor('abc', 3));
 function repeatStringNumTimesWithWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
+  const repeatArr = [];
+  if (num <= 0) {
```

Comment:
The same applies here. Because the loop condition for the while loop is tested before the loop body is executed, there is no need for an extra check beforehand.